app/models.py

```diff
@@ -514,6 +514,7 @@ class Service(BaseModel, Versioned):
     )
     active = db.Column(db.Boolean, index=False, unique=False, nullable=False, default=True)
     message_limit = db.Column(db.BigInteger, index=False, unique=False, nullable=False)
+    sms_daily_limit = db.Column(db.BigInteger, index=False, unique=False, nullable=False)
```

Comment:
So does `message_limit` now mean email limit?

Reply:
right now it's still the total notification limit - I haven't added any real code here, just added the new column to the database and model